subsys/testsuite/ztest/Kconfig

```diff
@@ -90,6 +90,7 @@ config ZTEST_PARAMETER_COUNT
 
 config ZTRESS
 	bool "Stress test framework"
+	depends on !SMP
```

Comment:
`!SMP` seems somewhat counterintuitive - for stressing, it would make sense to run a number of threads competing for resources, wouldn't it?

Reply:
need to remove it, when introduced ztress, initially it didn't run with SMP. It is fixed now. Will remove it.